src/current-refinements/__tests__/current-refinements.spec.ts

```diff
@@ -1,105 +1,152 @@
 import { createRenderer } from '../../../helpers/test-renderer';
+import { connectCurrentRefinements } from 'instantsearch.js/es/connectors';
 import { NgAisCurrentRefinements } from '../current-refinements';
 
-import { bem } from '../../utils';
-
-const cx = bem('CurrentRefinements');
-
 const defaultState = {
-  attributes: {},
-  clearAllClick: jest.fn(),
-  clearAllURL: jest.fn(),
   createURL: jest.fn(),
-  refined: jest.fn(),
-  refinements: [
+  refine: jest.fn(),
```

Comment:
Careful because we re-use multiples times the same object for the test. The `jest.fn()` are not reset so we might have some false positive in the future.

Reply:
you're right i'll make this a function everywhere to void theses traps 
```
const defaultState = () => ({...})
```